pgcli/main.py

```diff
@@ -418,6 +427,9 @@ def is_select(status):
         return False
     return status.split(None, 1)[0].lower() == 'select'
 
+def editor_command(sql):
+    return sql.strip().startswith('\e')
```

Comment:
I'm wondering if this can be converted to `sql.strip().endswith('\e')` which will handle the case when I type `SELECT * FROM users \e`. 

Although technically we're supposed to auto-populate the editor buffer with the preceding sql command `SELECT * FROM users` in this case, which will require more work.